document_multiple_records/document.py

```diff
@@ -51,6 +51,28 @@ def _name_get_resname(self, cr, uid, ids, obj, method, context):
                 data[attachment.id] = False
         return data
 
+    def _get_related_model_documents(
```

Comment:
@bwrsandman: _get_related_model_documents is the default for this new field; other reason, I am more comfortable in python.